packages/create-mc-app/src/process-options.ts

```diff
@@ -89,6 +110,7 @@ async function processOptions(
   });
   const entryPointUriPath = await getEntryPointUriPath(rl, options);
   const initialProjectKey = await getInitialProjectKey(rl, options);
+  const cloudIdentifier = await getCloudIdentifier(rl, options);
```

Comment:
Should the cloudIdentifier be prompted before or after the initialProjectKey?

Reply:
I think it's perfectly fine as it is 🙂